suite-native/module-send/src/components/FeeValueCard.tsx

```diff
@@ -0,0 +1,44 @@
+import { NetworkSymbol } from '@suite-common/wallet-config';
+import { isTestnet } from '@suite-common/wallet-utils';
+import { Card, HStack, VStack, Text } from '@suite-native/atoms';
+import { CryptoToFiatAmountFormatter, CryptoAmountFormatter } from '@suite-native/formatters';
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+
+type FeeValueCardProps = {
+    feeValue: string;
+    networkSymbol: NetworkSymbol;
+};
+
+const cardStyle = prepareNativeStyle(utils => ({
+    paddingVertical: utils.spacings.sp12,
+}));
+
+export const FeeValueCard = ({ feeValue, networkSymbol }: FeeValueCardProps) => {
+    const { applyStyle } = useNativeStyles();
+
+    return (
+        <Card style={applyStyle(cardStyle)}>
+            <HStack alignItems="center">
+                <Text variant="highlight">Transaction fee</Text>
```

Comment:
Please make it translatable. 

Also, in the design it is called "Mining fee", but I guess it is old copy? Please clarify in the description or figma comment what should be the current copy.